### PR TITLE
Updated: README with BlazingText and videogames to use prefix

### PR DESCRIPTION
**Pull request by djarpin**
Minor update to README to add a line for the new BlazingText algorithm.

I also slipped in a fix for the video game sales notebook based on this [issue](https://github.com/awslabs/amazon-sagemaker-examples/issues/166).  I tested the video game notebook.  It required a manual setup anyway, so as long as that direction is followed, it runs successfully on a SageMaker Notebook Instance.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ These examples provide quick walkthroughs to get you up and running with Amazon
 - [XGBoost for regression](introduction_to_amazon_algorithms/xgboost_abalone) predicts the age of abalone ([Abalone dataset](https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/regression.html)) using regression from Amazon SageMaker's implementation of [XGBoost](https://github.com/dmlc/xgboost).
 - [XGBoost for multi-class classification](introduction_to_amazon_algorithms/xgboost_mnist) uses Amazon SageMaker's implementation of [XGBoost](https://github.com/dmlc/xgboost) to classify handwritten digits from the MNIST dataset as one of the ten digits using a multi-class classifier. Both single machine and distributed use-cases are presented.
 - [DeepAR for time series forecasting](introduction_to_amazon_algorithms/deepar_synthetic) illustrates how to use the Amazon SageMaker DeepAR algorithm for time series forecasting on a synthetically generated data set.
+- [BlazingText Word2Vec](introduction_to_amazon_algorithms/blazingtext_word2vec_text8) generates Word2Vec embeddings from a large, cleaned text dump of Wikipedia articles using SageMaker's fast and scalable BlazingText implementation.
```
Comment (lukmis) [inline README.md]: The notebook says "small preprocessed dataset" (100MB) so please remove 'large' from description here. Potentially reword 'fast' into 'efficient' to match notebook.

File: introduction_to_amazon_algorithms/README.md [hunk 2]
```diff
@@ -14,4 +14,5 @@ These examples provide quick walkthroughs to get you up and running with Amazon
...
 - [XGBoost for regression](xgboost_abalone) predicts the age of abalone ([Abalone dataset](https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/regression.html)) using regression from Amazon SageMaker's implementation of [XGBoost](https://github.com/dmlc/xgboost).
 - [XGBoost for multi-class classification](xgboost_mnist) uses Amazon SageMaker's implementation of [XGBoost](https://github.com/dmlc/xgboost) to classifiy handwritten digits from the MNIST dataset as one of the ten digits using a multi-class classifier. Both single machine and distributed use-cases are presented.
-- [DeepAR for time series forecasting](deepar_synthetic) illustrates how to use the Amazon SageMaker DeepAR algorithm for time series forecasting on a synthetically generated data set.
\ No newline at end of file
+- [DeepAR for time series forecasting](deepar_synthetic) illustrates how to use the Amazon SageMaker DeepAR algorithm for time series forecasting on a synthetically generated data set.
+- [BlazingText Word2Vec](blazingtext_word2vec_text8) generates Word2Vec embeddings from a large, cleaned text dump of Wikipedia articles using SageMaker's fast and scalable BlazingText implementation.
```
Comment (lukmis) [inline introduction_to_amazon_algorithms/README.md]: Please update when updating the other place.

File: README.md (hunk 1 above)
Comment (djarpin) [reply README.md]: Good call.  Done.

File: README.md [hunk 3]
```diff
@@ -67,6 +68,12 @@ These examples focus on the Amazon SageMaker Python SDK which allows you to writ
 - [TensorFlow and TensorBoard](sagemaker-python-sdk/tensorflow_resnet_cifar10_with_tensorboard)
 - [Distributed TensorFlow](sagemaker-python-sdk/tensorflow_distributed_mnist)
 
+### Using Amazon SageMaker with Apache Spark
+
+These examples show how to use Amazon SageMaker for model training, hosting, and inference through Apache Spark using [SageMaker Spark](https://github.com/aws/sagemaker-spark). SageMaker Spark allows you to interleave Spark Pipeline stages with Pipeline stages that interact with Amazon SageMaker.
+
+- [MNIST with SageMaker Spark](sagemaker-spark/pyspark_mnist)
```
Comment (lukmis) [inline README.md]: Please update the link to use PySpark since the examples are pyspark.

Comment (djarpin) [reply README.md]: Done.

File: introduction_to_amazon_algorithms/README.md (hunk 2 above)
Comment (djarpin) [reply introduction_to_amazon_algorithms/README.md]: Done.

**Pull request merged**